todo-app/src/Task.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AppTask from './AppTask';
+import './index.css';
+
+export default class Task extends React.Component{
+
+  render() {
+  const{todoItem, toggleStatus, isEditing, onRemove, onEdit, onCancleEdit, onSaveEdit} = this.props;
+       return (
+            <li className="task-list">
+             <span> <input type = 'checkbox' checked = {todoItem.done} onChange = {() => toggleStatus(todoItem.id)}/></span>
+            <span> {todoItem.done? <span className = 'task-not-done'>
```

Comment:
You have something strange going on here. Why are you giving a class of `task-not-done` for tasks that _are_ done? I see this CSS declaration:

    .task-not-done {
      text-decoration:line-through;
    }

Seems you might have just named it incorrectly?